index.html

```diff
@@ -4,9 +4,146 @@
   <meta charset="UTF-8">
```

Comment:
- [ ] **You have multiple `<h1>` tags in your document. I will recommend you to stick with one.**

Although it has been determined that using multiple `h1` tags won't affect the ranking, it still advisable to use only `h1` tag if you don't require multiple `h1` tags. In this case, I think using one `h1` tag would be better. Please use one tag and structure the rest of your page with different `hx` tags. 

***Please implement the same in about.html**